### ISSUE TITLE
Default and stale branches

### ISSUE THREAD

**Issue by davelab6** [dalehenrich/filetree#189]
It seems there is recent activity on this project, but not in the default branch; and there are many merged/stale branches that could be pruned (easily with the github UI :)

**Comment by dalehenrich** [comment]
@davelab6m yes a number of the branches on this project need some loving attention, but the project is currently in a stable state with @ThierryGoubier doing the bulk of his work in support of Pharo5.0 and I am busy on other projects at the moment ...

**Comment by davelab6** [comment]
By stale I don't mean in need of more work; I mean, in need of deletion :) Is the "gemstone2.4" branch really the 'default' one?

**Comment by dalehenrich** [comment]
I understood that, but there are a number of platform branches that may 
appear to be stale, but aren't because there are multiple "primary" 
branches in this project and the github api doesn't make them obvious 
... so the work involved is thinking about each branch before clicking 
the button ... and to me that is "work" that is not at the top of my 
priority list ...

Dale

**Comment by dalehenrich** [comment]
well I've deleted the stale branches that were no brainers :)
